tests/integrationTests/dbCommandsTest.js

```diff
@@ -4,8 +4,21 @@ const DatabaseError = require('../../errors/DatabaseError');
 const { expect } = chai;
 
 const dbCommands = require('../../model/dbCommands');
+const dbCommandsAdmin = require('../../model/dbCommandsAdmin');
 const testDatabaseCommands = require('../../model/testDatabaseCommands');
 
+const adminUserTestDetails = {
+  username: 'AdminTestUser',
+  password: 'Secret123',
+  isAdmin: true,
+};
+
+const userTestDetails = {
+  username: 'TestUser',
+  password: 'Secret456',
+  isAdmin: 0,
```

Comment:
is it better to put false here to keep it consistent?

Reply:
done